<launch>
    <!-- lane detection camera -->	
	<node name="perspective_transform_node" pkg="capstone_utils" type="perspective_transform_node" output="screen">
		<param name="source_name" type="string" value="usb_cam_1/image_raw/compressed"/>
	</node>
</launch>
